tests/test_shipment.py

```diff
@@ -216,10 +216,10 @@ def test_smartrate(vcr):
 
     smartrates = shipment.get_smartrates()
     assert shipment.rates[0]['id'] == smartrates['result'][0]['id']
-    assert smartrates['result'][0]['time_in_transit']['percentile_50'] == 2
+    assert smartrates['result'][0]['time_in_transit']['percentile_50'] == 1
     assert smartrates['result'][0]['time_in_transit']['percentile_75'] == 2
-    assert smartrates['result'][0]['time_in_transit']['percentile_85'] == 3
+    assert smartrates['result'][0]['time_in_transit']['percentile_85'] == 2
```

Comment:
Assuming this is just that the results changed based on the shipment defaults at time of call?

Reply:
Yeah, re-recorded the cassette for good measure, these can change on each run.